### PR TITLE
View application details

### PR DESCRIPTION
**Pull request by timpeat**
## Description of change
Adds a show page sections for application details in review

## Link to relevant ticket

[CRIMRE-28](https://dsdmoj.atlassian.net/browse/CRIMRE-28)
[CRIMRE-29](https://dsdmoj.atlassian.net/browse/CRIMRE-29)
[CRIMRE-30](https://dsdmoj.atlassian.net/browse/CRIMRE-30)
[CRIMRE-41](https://dsdmoj.atlassian.net/browse/CRIMRE-41)

## Screenshots of changes (if applicable)
Screenshot 2022-10-21 at 17 47 47

### CODE REVIEW

File: app/models/crime_application.rb [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+  end
 
-      {
-        id: source['id'],
-        applicant_name: applicant_name,
-        laa_reference: source['application_reference'],
-        received_at: source['application_start_date']
-      }
-    end
+  def common_platform?
```
Comment (willmcb) [inline app/models/crime_application.rb:23]: We can infer this from user input from apply

File: app/models/person.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Person < ApplicationStruct
+  attribute :first_name, Types::String
+  attribute :last_name, Types::String
+  attribute :address, Address
+  attribute :telephone_number, Types::PhoneNumber
+  attribute? :national_insurance_number, Types::String
+  attribute? :date_of_birth, Types::JSON::Date
+  attribute? :correspondence_address
```
Comment (willmcb) [inline app/models/person.rb:8]: ?

File: app/views/crime_applications/_overview.html.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <%= overview.means_type %>
+    </dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      <%= t(:common_platform, scope: 'crime_applications.labels') %>
+    </dt>
+    <dd class="govuk-summary-list__value">
+      <%= overview.common_platform? ? 'yes' : 'no'%>
```
Comment (willmcb) [inline app/views/crime_applications/_overview.html.erb:29]: use i18n

**Review by willmcb — APPROVED** [review]
LGTM 👍

File: app/models/person.rb (hunk 2 above)
Comment (timpeat) [reply app/models/person.rb:8]: fixed

File: app/views/crime_applications/_overview.html.erb (hunk 3 above)
Comment (timpeat) [reply app/views/crime_applications/_overview.html.erb:29]: done

**Pull request merged**